### PR TITLE
feat: add grade value to degreed v2

### PR DESCRIPTION
**Pull request by katrinan029**
## Description
The Degreed v2 API incorporated the option to add the value a learner's grade. 
https://developer.degreed.com/reference/post_api-v2-completions

## Solution
Add the `percentile` field to support passing the grade value.

[JIRA](https://2u-internal.atlassian.net/browse/ENT-5809)

**Merge checklist:**
- [ ] Any new requirements are in the right place (do **not** manually modify the `requirements/*.txt` files)
    - `base.in` if needed in production but edx-platform doesn't install it
    - `test-master.in` if edx-platform pins it, with a matching version
    - `make upgrade && make requirements` have been run to regenerate requirements
- [ ] `make static` has been run to update webpack bundling if any static content was updated
- [ ] `./manage.py makemigrations` has been run
    - Checkout the [Database Migration](https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations) Confluence page for helpful tips on creating migrations.
    - *Note*: This **must** be run if you modified any models.
      - It may or may not make a migration depending on exactly what you modified, but it should still be run.
    - This should be run from either a venv with all the lms/edx-enterprise requirements installed or if you checked out edx-enterprise into the src directory used by lms, you can run this command through an lms shell.
        - It would be `./manage.py lms makemigrations` in the shell.
- [ ] [Version](https://github.com/openedx/edx-enterprise/blob/master/enterprise/__init__.py) bumped
- [ ] [Changelog](https://github.com/openedx/edx-enterprise/blob/master/CHANGELOG.rst) record added
- [ ] Translations updated (see docs/internationalization.rst but also this isn't blocking for merge atm)

**Post merge:**
- [ ] Tag pushed and a new [version](https://github.com/openedx/edx-enterprise/releases) released
    - *Note*: Assets will be added automatically. You just need to provide a tag (should match your version number) and title and description.
- [ ] After versioned build finishes in [GitHub Actions](https://github.com/openedx/edx-enterprise/actions), verify version has been pushed to [PyPI](https://pypi.org/project/edx-enterprise/)
    - Each step in the release build has a condition flag that checks if the rest of the steps are done and if so will deploy to PyPi.
    (so basically once your build finishes, after maybe a minute you should see the new version in PyPi automatically (on refresh))
- [ ] PR created in [edx-platform](https://github.com/openedx/edx-platform) to upgrade dependencies (including edx-enterprise)
    - This **must** be done after the version is visible in PyPi as `make upgrade` in edx-platform will look for the latest version in PyPi.
    - Note: the edx-enterprise constraint in edx-platform **must** also be bumped to the latest version in PyPi.

### CODE REVIEW

File: integrated_channels/degreed2/client.py [hunk 1]
```diff
@@ -114,6 +114,7 @@ def create_course_completion(self, user_id, payload):
         Returns: status_code, response_text
         """
         json_payload = json.loads(payload)
+        json_payload['data']['attributes']['percentile'] = json_payload['data']['attributes']['percentile'] * 100
```
Comment (alex-sheehan-edx) [inline integrated_channels/degreed2/client.py:117]: 🤔 Wondering if this can go in the exporter

**Review by alex-sheehan-edx — APPROVED** [review]
If you move the data transformation in your client bit to the degreed2 exporter then you can also write a test to make sure grade percent comes out of the degreed2 `get_learner_data_records` method!  Once you get that, this looks great to me

File: integrated_channels/degreed2/client.py (hunk 1 above)
Comment (katrinan029) [reply integrated_channels/degreed2/client.py]: good suggestion - I think it's a little cleaner moving this into the exporter as well!

File: integrated_channels/degreed2/exporters/learner_data.py [hunk 2]
```diff
@@ -32,6 +32,7 @@ def get_learner_data_records(
 
         If no remote ID can be found, return None.
         """
+        percent_grade = kwargs.get('grade_percent', None)
```
[the remark this answers is not in the document]
Comment (katrinan029) [reply integrated_channels/degreed2/exporters/learner_data.py]: Done!

**Pull request merged**